go.mod

```diff
@@ -89,6 +89,13 @@ require (
 	gopkg.in/guregu/null.v4 v4.0.0
 )
 
+require (
+	github.com/btcsuite/btcd/btcec/v2 v2.2.0 // indirect
+	github.com/go-errors/errors v1.4.1 // indirect
+	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6 // indirect
+	github.com/tendermint/btcd v0.1.1 // indirect
+)
```

Comment:
I don't know why go mod does this for some people but let's keep one block for `indirect`.
What version of go do you have installed?

Reply:
```
$ go version
go version go1.18 darwin/arm64

$ cat .tool-versions
golang 1.18
mockery 2.13.0-beta.1
nodejs 16.13.2
postgres 13.3
ginkgo 2.1.4
```
For a while I was using 1.18.2, but I kept getting a lot of errors in vim because it didn't match what was in the `.tool-versions`, so I switched back to 1.18 a few days ago.

Reply:
hmmmm if we have a stale `.tool-versions` file then we should update it. It is important to stay on the latest minor release - 1.18.4 in this case.

Reply:
So glad to hear that!  This has been a pain point for a while, I assumed there was some reason why we were keeping it at 1.18.  I'll add it to this PR